File: odd_even_day.py
from datetime import datetime, timezone
def odd_or_even_day(timestamp):
    date_object = datetime.fromtimestamp(timestamp / 1000, tz = timezone.utc)
    day = date_object.day
    print(day)
    return 'odd' if day % 2 != 0 else 'even'
#Demo test !!!Eslatma bu UTC 0 dagi time zona ishlatilgan 
print(odd_or_even_day(6739456780000))#Output: odd
print(odd_or_even_day(86400000))#Output: even

File: odd_even_day.js
function oddOrEvenDay(timestamp) {
	const date = new Date(timestamp)
	const day = date.getUTCDate()

	return day % 2 == 0 ? 'even' : 'odd'
}
//Demo test !!! UTC time zonasi uchun
console.log(oddOrEvenDay(1769472000000))
console.log(oddOrEvenDay(1769444440000))
